src/test/java/tr/com/srdc/cda2fhir/testutil/JoltUtil.java

```diff
@@ -135,29 +135,6 @@ public void copyReferences(Map<String, Object> joltResult) {
 		}
 	}
 
-	private static class OrganizationInfo extends ResourceInfo {
```

Comment:
looks like you changed some stuff with organization and patient as well, was this intentional? Were you fixing some of that work left to do?